src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -196,9 +208,18 @@ public class OHTable implements HTableInterface {
     public OHTable(Configuration configuration, String tableName) throws IOException {
```

Comment:
this passed configuration from OHTablePool will be shared by multi-thread, which will conflict when OHTablePool.getTable concurrently.